qiskit/circuit/parameterexpression.py

```diff
@@ -526,6 +526,12 @@ def is_real(self):
                 return False
         return True
 
+    def get_sympy_expr(self):
+        """Return symbolic expression from sympy"""
+        from sympy import sympify
+
+        return sympify(self._symbol_expr)
+
```

Comment:
We use `symengine` preferentially here, because `sympy` can be very slow.  We probably don't want to throw that away.

Reply:
Included code to use `symengine` if available.